QuoteGeneration/buildenv.mk

```diff
@@ -194,6 +194,16 @@ CXXFLAGS += $(COMMON_FLAGS)
 # Enable the security flags
 COMMON_LDFLAGS := -Wl,-z,relro,-z,now,-z,noexecstack
 
+# Enable build static library
+GEN_STATIC ?= 0
+GEN_DYNAMIC ?= 1
```

Comment:
One question, is the mode `GEN_STATIC=1 GEN_DYNAMIC=0` required for you? Currently both the installer scripts and DCAP sample code only support dynamic libs.  Based on an internal code review, we think it seems better to always enable the dynamic lib generation and make static lib generation as optional. Otherwise, to fully support the modes, we need to change the installer scripts and sample code.

Reply:
OK, I will remove the option `GEN_DYNAMIC`

Reply:
Done. Please check.